### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by lonasozo** [community/community#30494] [Copilot Conversations]
Got this error, clicking on "Show output Log" does not show anything. 

![image](https://user-images.githubusercontent.com/34058834/186086255-e9cb362b-8ecd-4947-8942-70652d276006.png)

Actually I have activated trial: 
![image](https://user-images.githubusercontent.com/34058834/186086514-4c680d6a-1836-4b89-9cff-7beb21499668.png)

EDIT: installing nightly solve the problem

**Comment by NikhilSehgal123** [comment]
I have paid for GitHub Copilot but is is not activating!! Can someone from the community please help??

**Reply by lonasozo** [reply]
Install the nightly version could temporarily solve the problem (at least in my case)
![image](https://user-images.githubusercontent.com/34058834/186087896-deed1304-1cab-4673-bb86-cfd95049d11b.png)

**Reply by NikhilSehgal123** [reply]
Thanks, but unfortunately it didn't work for me

**Reply by davecheney** [reply]
@NikhilSehgal123 I see your account active and receiving completions as of 1 hour ago. Is your issue solved?

**Reply by NikhilSehgal123** [reply]
@davecheney no it's not solved. I attempted to generate a completion one hour ago but I keep getting the following erro
![Screenshot 2022-08-23 at 09 40 36](https://user-images.githubusercontent.com/49030309/186113358-b478912d-60ba-45ce-9cb2-2edd29716fd7.png)
r

**Reply by NikhilSehgal123** [reply]
@davecheney any idea how to solve this?

**Reply by davecheney** [reply]
Have you tried upgrading your copilot extension?

**Reply by NikhilSehgal123** [reply]
@davecheney yes already tried that

**Reply by davecheney** [reply]
You are running version 1.31, the latest is version 1.42. https://marketplace.visualstudio.com/items?itemName=GitHub.copilot
